template/android/build.gradle

```diff
@@ -2,9 +2,9 @@
 
 buildscript {
     ext {
-        buildToolsVersion = "28.0.3"
+        buildToolsVersion = "29.0.2"
         minSdkVersion = 16
-        compileSdkVersion = 28
+        compileSdkVersion = 29
         targetSdkVersion = 28
```

Comment:
@dulmandakh 

It seems you forgot to bump tagetSdkVersion(s) to 29 too. AFAIK `targetSdkVersion` should not be less than `compileSdkVersion`.

I read [the above comment](https://github.com/facebook/react-native/pull/26909#issuecomment-564411531), are you want to merge these changes before bumping `targetSdkVersion`

Reply:
@SaeedZhiany - If you check my (collapsed) comment from Oct 18, 2019 above, you can see I had the same question, and @dulmandakh already responded. It sounds like the update to `targetSdkVersion` will come at a later point.

Reply:
Yeah, I saw that and my question is why we don't wait until we can bump targetSdkVersion too, then merge it. usually Android studio shows a warning (or error) message on Gradle file when targetSdkVersion less than compileSdkVersion. (I can't remember what the message is though)